README.md

```diff
@@ -150,6 +150,19 @@ This article explains in detail how to use the UniFi Site Export Wizard
 to quickly and easily export sites from one Controller
 (including configuration and devices) to another (e.g., this add-on).
 
+## Using Advanced Configuration
+
+You can use the advanced configuration via config.gateway.json as outlined 
+in this tutorial from Ubiquiti:
+
+<https://help.ubnt.com/hc/en-us/articles/215458888-UniFi-USG-Advanced-Configuration>
+
+The folder to copy your config.gateway.json is in `/backup/unifi/config`. 
+You will have to choose your site you wish to configurate. (Standard = Default)
```

Comment:
Small typo: *the* site you wish to *configure*

Reply:
Thank you :) However, i do not know how i can make these changes within this pull request... any tipps?

Reply:
Make the changes to your local copy and push, the pull request will get updated.